extensions/amp-story-interactive/0.1/amp-story-interactive-binary-poll.css

```diff
@@ -117,12 +117,12 @@
   margin: 0 !important;
   line-height: 1.3em !important;
   width: 4.42em !important;
-  height: 1.3em !important;
   display: flex !important;
   align-items: center !important;
   justify-content: center !important;
   transition: transform var(--i-amphtml-interactive-animation-time)
     var(--i-amphtml-interactive-ease-out-curve) !important;
+  height: 2.6em !important;
```

Comment:
I wonder if this is needed.
Is the `max-height` specified on the child sufficient for this?

Reply:
Not enough since the actual text could have 1 or 2 lines, which changes the height of the text. But we want it to occupy always the same size on the parent div, so the container needs to have a predetermined height (it used to be 1.3 but I changed it to 2.6 since that's the height of 2 lines of text)

Reply:
Got it! I see how this is working with the `translateY` now.

Reply:
Yeah! When the title shrinks it leaves a big gap between it and the percentage (which was showing literally on the bottom edge when the title had 2 lines). So everything was working on 1 line but not 2 lines. Not sure how I missed this, or maybe it was introduced in another PR that I'm not aware of... Anyways this will fix it